jdaviz/core/template_mixin.py

```diff
@@ -4486,7 +4486,10 @@ def _on_glue_value_changed(self, value):
     def unmix_state(self, new_value=None):
         if new_value is None:
             new_value = self.value
-        self._on_value_changed({'new': new_value})
+        if new_value != self.value:
+            self.value = new_value
+        else:
+            self._on_value_changed({'new': new_value})
```

Comment:
If `new_value == self.value`, why is `_on_value_changed` needed?

Reply:
to propagate to all the linked states (this happens when clicking to unmix the state, `self.value` is the nominal value (of the first linked state) and we want to propagate that to all other linked glue states)

Reply:
(see the description above for the scenario where this makes a difference, probably could do the same by forcing a send_state, but that would then call `_on_value_changed` twice)